### PR TITLE
[Tests-Only] Split webUiSharingInternalUsers suite into smaller suites

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
move some of the `webUiSharingInternalUsers` feature files into a separate suite folder
- webUiSharingInternalUsers
- webUiSharingInternalUsersCollaborator
- webUiSharingInternalUsersShareWithPage

## Related Issue
https://github.com/owncloud/web/issues/4876

## How Has This Been Tested?
- test environment:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersToShare2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14210/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersToShare2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14216/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersToShare2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14220/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersToShare2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14231/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersToShare2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14240/

<p>





</p></details>

File: .drone.star [hunk 1]
```diff
@@ -78,7 +78,11 @@ config = {
 				'webUISharingInternalGroupsToRoot': 'SharingInternalGroupsRoot',
 				'webUISharingInternalGroupsToRootEdgeCases': 'SharingInternalGroupsRootEdgeCases',
 				'webUISharingInternalGroupsToRootSharingIndicator': 'SharingInternalGroupsRootSharingIndicator',
-				'webUISharingInternalUsers': 'SharingInternalUsers',
+				'SharingInternalUsers': [
```
Comment (dpakach) [inline .drone.star:81]:
```suggestion
				'webUISharingInternalUsers': [
```

**Comment by HariBhandari07** [comment]
Except for the naming of the suite as mentioned in the above review all looks good. :+1:

**Comment by kiranparajuli589** [comment]
lgtm 👍🏼

**Review by individual-it — APPROVED** [review]

**Pull request merged**